### ISSUE TITLE
Errors with 'Dogeforia' and its associated classes

### ISSUE THREAD

**Issue by Flashman2000** [MechanicalMemes/DogeCV#8]
Imports statements in 'Dogeforia' aren't working. Problem was resolved by moving Dogeforia to TeamCode

**Comment by victoryforphil** [comment]
This appears to be an issue in which DogeCV isn't properly implementing FtcRobotController. Looking into this now.

**Comment by dadosch** [comment]
If you add 
```
dependencies {
    implementation project(':DogeCV')
}
```
to TeamCode's build.gradle it seems to compile

**Comment by victoryforphil** [comment]
Looks like I managed to fix this is 2018.1 by directly importing the RobotCore lib. Until the official release please either move Dogeforia to TeamCode or Change dependencies under the DogeCV build.gradle to this:
```dependencies {
    implementation fileTree(include: ['*.jar'], dir: 'libs')
    implementation project(path: ':openCVLibrary3')
    implementation (name: 'RobotCore-release', ext: 'aar')
}```

**Comment by victoryforphil** [comment]
@lindenrobots7031 Yea however if my fix doesn't work ill reopen it.
